runtime/lib/component/audio-mix.js

```diff
@@ -0,0 +1,77 @@
+var logger = require('logger')('audioMix')
+var AudioManager = require('@yoda/audio').AudioManager
+var GlobalConfig = {
+  scale: 0.3
+}
+
+try {
+  GlobalConfig = require('/etc/yoda/global-config.json')
```

Comment:
Where is the config?

Reply:
should be `system-config.json`. already fixed.